### ISSUE TITLE
Failed to minify `@nteract/styles/lib/styles.js` with create-react-app

### ISSUE THREAD

**Issue by tlyon3** [nteract/archived-desktop-app#2655]
When building a react application using create-react-app, running `npm run build` returns a “Failed to minify” error resulting from `@nteract/styles/lib/styles.js:3`.

**Comment by tlyon3** [comment]
Here is the output of my log:

```
➜  client git:(master) ✗ npm run build

> client@0.3.0 build /Users/tlyon3/Work/quantecon/repos/Bookshelf/client
> react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/@nteract/styles/lib/styles.js:3 

Read more here: http://bit.ly/2tRViJ9

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! client@0.3.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the client@0.3.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tlyon3/.npm/_logs/2018-03-09T04_48_02_441Z-debug.log
```

**Comment by rgbkrk** [comment]
Ah, I see. [`create-react-app` won't allow template literals](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify). It's bombing on:

```
module.exports = `
:root {
```

**Comment by rgbkrk** [comment]
We'll have to change our babel configuration to make sure template literals are transpiled.

**Comment by rgbkrk** [comment]
In the _next_ release of `create-react-app` it looks like they'll transpile dependencies, based on https://github.com/facebook/create-react-app/pull/3776 being on their `next` branch since January (it's not on `master`, so not shipped).

**Comment by rgbkrk** [comment]
Ok, so our `babel` config _would_ create the right output. The issue is that this file in particular is generated javascript:

```
// Generate themeable css. Overwrite the whole file for the first write.
fs.writeFileSync(
  CSS_PATH,
  "// This file is auto-generated. See buildStyles.js for details.\n\n"
);
fs.appendFileSync(CSS_PATH, "module.exports = `\n:root {\n");
lodash.toPairs(declarations).forEach(([cssProperty, cssValue]) => {
  fs.appendFileSync(CSS_PATH, `  ${cssProperty}: ${cssValue};\n`);
});
fs.appendFileSync(CSS_PATH, "}\n`;\n");
```

Instead we have to create one long string there instead of using a \`.

**Comment by rgbkrk** [comment]
Closed as of #2674.
